Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -63,6 +63,32 @@ python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --
 * `--topology-pools`: (optional) Comma-separated list of topology-constrained rbd pools
 * `--topology-failure-domain-label`: (optional) K8s cluster failure domain label (example: zone, rack, or host) for the topology-pools that match the ceph domain
 * `--topology-failure-domain-values`: (optional) Comma-separated list of the k8s cluster failure domain values corresponding to each of the pools in the `topology-pools` list
+* `--config-file`: Path to the configuration file, Priority: command-line-args > confie.ini values > default values
+
+### Config-file
+
+Use the config file to set the user configuration file, add the flag `--config-file` to set the file path.
+
+Example:
+
+```console
+/config.ini
+
```

Comment:
Let's not include this in the text here. Some users might assume that this is a required line, and I think it would likely cause a parse error in the code.

`/config.ini`

```
[Configurations]
...
```